libs/ngxtension/reduce-array/src/reduce-array.ts

```diff
@@ -0,0 +1,9 @@
+import { map } from 'rxjs';
+
+export const reduceArray = <T, R>(
```

Comment:
suggestion: for `reduce`, it's better to make `R = T` so if the consumers don't pass in initial value, then the return type is the type of the item in the Array. I.e: `nums.reduce((a, c) => a + c)`

```suggestion
export const reduceArray = <T, R = T>(
```

Reply:
yeap, also the initialValue should be optional. 
ill add it tomorrow since i also want to include unit test for that case